examples/todo/src/__tests__/acceptance/todo.acceptance.ts

```diff
@@ -222,6 +222,110 @@ describe('TodoApplication', () => {
     expect(response.body).to.have.length(2);
   });
 
+  it('queries todos with string-based order filter', async () => {
+    const todoInProgress = await givenTodoInstance({
+      title: 'go to sleep',
+      isComplete: false,
+    });
+
+    const todoCompleted = await givenTodoInstance({
+      title: 'wake up',
+      isComplete: true,
+    });
+
+    const todoCompleted2 = await givenTodoInstance({
+      title: 'go to work',
+      isComplete: true,
+    });
+
+    await client
+      .get('/todos')
+      .query({filter: {order: 'title DESC'}})
+      .expect(200, [
+        toJSON(todoCompleted),
+        toJSON(todoCompleted2),
+        toJSON(todoInProgress),
+      ]);
```

Comment:
Nitpick - feel free to ignore.

The code can be simplified as follows.

```suggestion
      .expect(200, toJSON([
        todoCompleted,
        todoCompleted2,
        todoInProgress,
      ]));
```

Same comment applies to other new tests too.